src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubSettings.cs

```diff
@@ -111,8 +117,15 @@ public static DistributedPubSubSettings Create(Config config)
         /// <param name="gossipInterval">TBD</param>
         /// <param name="removedTimeToLive">TBD</param>
         /// <param name="maxDeltaElements">TBD</param>
+        /// <param name="sendToDeadLettersWhenNoSubscribers">When a message is published to a topic with no subscribers send it to the dead letters.</param>
         /// <exception cref="ArgumentException">TBD</exception>
-        public DistributedPubSubSettings(string role, RoutingLogic routingLogic, TimeSpan gossipInterval, TimeSpan removedTimeToLive, int maxDeltaElements)
+        public DistributedPubSubSettings(
```

Comment:
Probably need to add a constructor overload here to avoid a breaking change that will have real binary compatibility impact... I'll do that in a subsequent PR